lua/astrocommunity/pack/python/init.lua

```diff
@@ -88,8 +89,15 @@ return {
         },
       },
     },
-    opts = {},
-    cmd = "VenvSelect",
+    opts = {
+      -- options = {
+      --   debug = true,
+      -- },
+    },
+    command = {
```

Comment:
You can check the docs for what options are available for lazy loading: https://lazy.folke.io/spec#spec-lazy-loading

Reply:
@rodrigopmatias Sorry for not being clear enough. But you removed the `cmd` that was used before and added `command` which doesn't exist. The `lazy` is not needed, since `ft` and `cmd` indicate already that it should be lazy.